package.json

```
@@ -18,12 +18,13 @@
 		"lint:styles": "stylelint packages/**/styles.ts",
 		"prepare": "husky install",
 		"release": "yarn clean && yarn build && changeset publish",
-		"storybook": "start-storybook -p 6006",
 		"test": "nx run-many --all --target=test",
 		"test:coverage": "nx run-many --target=test --all --coverage",
 		"tsc": "tsc",
 		"validate": "npm-run-all tsc lint test build e2e",
-		"e2e": "nx run-many --target=e2e --all"
+		"e2e": "nx run-many --target=e2e --all",
+		"storybook": "start-storybook -p 6006",
+		"build-storybook": "build-storybook"
```

think you need this name for the chromatic workflow to run correctly (or update the workflow)

```suggestion
		"build:storybook": "build-storybook"
```

---

ah - looks like we already have a `build:storybook` here: https://github.com/guardian/source/pull/1608/files/49ec530f5c1622432dff60004178d7dd8c491fce#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R12 

so I think safe to remove this line completely